src/data/extensions/extensions.js

```diff
@@ -238,7 +238,7 @@ const extensions = [
   },
   {
     name: 'xk6-influxdbv2',
-    description: 'A k6 extension output to influxdb',
+    description: 'Export results to influxdb',
     url: 'https://github.com/li-zhixin/xk6-influxdbv2',
```

Comment:
We actually just added an "official" InfluxDB v2 extension here: https://github.com/grafana/xk6-output-influxdb
Perhaps, this link should be updated too?

Reply:
Thanks! Fixed by https://github.com/grafana/k6-docs/pull/474/commits/66cf51d757aca302bdc7186516172aa22b0fa9d0